Dockerfile

```diff
@@ -325,6 +329,7 @@ ARG DOCKER_BUILDTAGS
 ENV DOCKER_BUILDTAGS="${DOCKER_BUILDTAGS}"
 WORKDIR /go/src/github.com/docker/docker
 VOLUME /var/lib/docker
+VOLUME /home/unprivilegeduser/.local/share/docker
```

Comment:
This is the only one I was a bit on the fence on; considering if it would be better if we make this a `-v /home/unprivilegeduser/.local/share/docker` at runtime instead

Reply:
Do we do that for `VOLUME /var/lib/docker` as well?

Reply:
No, I don't think we do; I think the main reason for that being there was that otherwise the container would be (largely) non-functional (unless `vfs` is used). Downside of these `VOLUME` declarations is that now at least _two_ anonymous volumes are created each time.

Not a big deal.